schemas/parser-metadata/parser.schema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "Parser process metadata",
+  "description": "Metadata about a document or its processing which has been generated or collated as a result of the Find Case Law parsing process.",
+  "properties": {
+    "documentType": {
+      "title": "Type of document",
+      "description": "Must be one of the document types supported by Find Case Law.",
+      "type": "string",
+      "enum": ["judgment", "pressSummary"]
+    },
+    "uri": {
+      "title": "Document URI",
+      "type": "string",
+      "format": "uri"
+    },
+    "court": {
+      "title": "Court",
+      "type": "string"
```

Comment:
Worth mentioning that this is an enumeration, even if we don't do that yet, and maybe point at where that's defined